code/modules/mob/dead/observer/observer.dm

```diff
@@ -49,6 +49,7 @@ var/list/image/ghost_darkness_images = list() //this is a list of images for thi
 			else
 				name = random_unique_name(gender)
 
+		body.mind.ghost = TRUE
```

Comment:
Bollocks. Will try to fix, but I'm surprised by this. I did a lot of testing on my local machine using my Byond account and a guest one